### PR TITLE
Add code coverage

### PR DESCRIPTION
**Pull request by zbraniecki**
Follow up from #90.

### CODE REVIEW

**Comment by zbraniecki** [comment]
Here's the example output - https://coveralls.io/github/zbraniecki/icu4x

File: .github/workflows/coverage.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    steps:
+      - uses:                   actions/checkout@v2
+
+      - uses:                   actions-rs/toolchain@v1
+        with:
+          profile:              minimal
+          toolchain:            nightly
+          override:             true
+
+      - uses:                   actions-rs/cargo@v1
```
Comment (echeran) [inline .github/workflows/coverage.yml:24]: Interesting, there's an `actions-rs` project containing Rust-specific actions.  The Build & Test action for CI that we have just uses a vanilla `cargo test` -- is it okay to leave it as it is for now?

Comment (zbraniecki) [reply .github/workflows/coverage.yml:24]: Yeah, I assume that gh actions are new enough that the ecosystem is fairly dynamic. We can refactor and unify once it stabilizes.

**Comment by zbraniecki** [comment]
I need someone with ownership access over `unicode-org` to go to https://coveralls.io/ and add `unicode-org/icu4x` as a repo. @sffc ?

**Pull request merged**